.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,8 @@
+<gradleEnterprise>
+  <buildScan>
+    <termsOfService>
+      <url>https://gradle.com/terms-of-service</url>
```

Comment:
As far as I understand this is not compatible with an Apache project.

Reply:
I've asked on legal-discuss@a.o for advice. 
If it's not ok.

Reply:
Legal is ok https://mail-archives.apache.org/mod_mbox/www-legal-discuss/202007.mbox/browser (Reply from Roy Fielding may not be yet in archive) 
Waiting on infra to agree to the TOS and we are set.